components/automate-cs-oc-erchef/habitat/default.toml

```diff
@@ -40,6 +40,10 @@ reindex_batch_size=10
 reindex_sleep_min_ms=50
 reindex_sleep_max_ms=2000
 reindex_item_retries=3
+search_provider="elasticsearch"
```

Comment:
should make it to 
search_provider="opensearch" , to make it Compatible for opensearch